src/forms/RadioButtonGroup.jsx

```diff
@@ -24,46 +25,26 @@ export default class RadioButtonGroup extends PureComponent {
 		}
 	}
 
-	handleChange(event) {
-		const { onChange } = this.props;
+	handleChange(e) {
+		this.props.onChange && this.props.onChange(e);
 
 		this.setState({
-			selectedValue: event.target.value,
+			selectedValue: e.target.value,
 		});
-
-		if (onChange) {
-			onChange(event);
-		}
 	}
 
 	render() {
-		const {
-			direction,
-			switchDirection,
-			children,
-			name,
-			className,
-			wrap,
-			justifyItems,
-		} = this.props;
-
-		const switchDirectionAttr = switchDirection ? { switchDirection } : '';
+		const { children, className, direction } = this.props;
 
 		return (
-			<Flex
-				direction={direction}
-				{...switchDirectionAttr}
-				className={className}
-				wrap={wrap}
-				justifyItems={justifyItems}
-			>
-				{React.Children.map(children, option => (
+			<Flex direction={direction} className={className}>
```

Comment:
I intentionally removed the `wrap` and `justifyItems` props because:

1) They're not documented anywhere
2) Mup-web, pro-web, and MWC do not use those props.